xikolo-common/Data/Model/Course.swift

```diff
@@ -83,6 +83,14 @@ class Course : BaseModel {
 
 }
 
+extension Course : DynamicSort {
+
+    func computeOrder() {
+        self.order = NSNumber(value: abs(start_at?.timeIntervalSinceNow ?? TimeInterval.infinity))
```

Comment:
It's already in that line; it gets set to `infinity` and will end up at the bottom of the list.